Source/FolioReaderChapterListCell.swift

```diff
@@ -9,34 +9,50 @@
 import UIKit
 
 class FolioReaderChapterListCell: UITableViewCell {
-    var indexLabel = UILabel()
-    
+
+    var indexLabel 			: UILabel?
```

Comment:
This isn't a good practice (a lot of space)

https://github.com/raywenderlich/swift-style-guide